testing/e2e/suite/comand_line_integration.go

```diff
@@ -0,0 +1,215 @@
+//go:build integration
```

Comment:
Type in filename. should be `command_line_integration.go` ?

Reply:
correct, fixed